lib/pages/home.dart

```diff
@@ -97,7 +101,29 @@ class _HomeState extends State<Home> {
                     _buildAddButton(Icons.label_outline, "Category",
                         () => addNewCategory(context)),
                     //_buildAddButton(Icons.done_outline, "Progress", null),
-                    _buildAddButton(Icons.share, "Share", null),
+                    _buildAddButton(
+                        Icons.share,
+                        "Share",
+                        () => EasyDialog(height: 200, contentList: [
```

Comment:
Instead of the dialog, just give the option to share the app for now. This feature might be replaced by something else in a future build.